src/Python/Tutorial/Advanced/color_map_optimization.py

```diff
@@ -0,0 +1,58 @@
+# Open3D: www.open3d.org
+# The MIT License (MIT)
+# See license file or visit www.open3d.org for details
+
+from py3d import *
+from trajectory_io import *
+import os, sys
+sys.path.append("../Utility")
+from common import *
+
+path = "[set_this_path_to_fountain_dataset]"
```

Comment:
I didn't include dataset for ColorMapOptimization. Instead, I had added google drive link to download the dataset. The file size for this tutorial is about 30MB (including mesh, 33 color and depth image pairs), which is not need to be maintained in Github repo.